docs/source/programming_resources/shared/configuring_android/Configuring-Your-Android-Devices.rst

```diff
@@ -0,0 +1,545 @@
+Configuring your Android Devices
+================================
+
+What Needs to Be Configured for My Control System?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Control Hub Configuration 
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Teams who are using a Control Hub with the integrated Robot Controller
+will only need to configure a single smartphone for use as a Driver
+Station. The process is as follows: 
+
+*  Rename the smartphone to "<TEAM NUMBER>-DS" (where <TEAM NUMBER> is replaced by your team number). 
+*  Install the Driver Station (DS) app onto the Driver Station phone. (The DS app is pre-installed on the REV Driver Hub.) 
+*  Put your phone into Airplane Mode (with the WiFi radio still on). 
+*  Pair (i.e., wirelessly connect) the Driver Station to the Control Hub.
+
+.. image:: images/ControlHubAndPhone.jpg
+   :align: center
+
+|
+
+References to the Driver Station smartphone may instead apply to the
+`REV Driver
+Hub <https://docs.revrobotics.com/rev-control-system/driver-hub-gs>`__,
+which is preloaded with the FTC Driver Station (DS) app.
+
+**IMPORTANT NOTE:** Eventually the Control Hub will need be renamed so
+that its name complies with Game Manual rule <RS01>, but for now we will
+use the Control Hub with its default name. You can learn how to manage a
+Control Hub (and modify its name, password, etc.) in :doc:`this
+tutorial <../managing_control_hub/Managing-a-Control-Hub>`.
+
+Two Android Smartphone Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Teams who have two smartphones and are not using a Control Hub will need
+to configure one smartphone for use as a Robot Controller and a second
+smartphone for use as a Driver Station. The process is as follows, 
+
+*  Rename one smartphone to "<TEAM NUMBER>-RC" (replace <TEAM NUMBER> with your team number). 
+*  Install the Robot Controller app onto the Robot Controller phone. 
+*  Rename a second smartphone to "<TEAM NUMBER>-DS" (where <TEAM NUMBER> is replaced by your team number). 
```

Comment:
```
`<TEAM NUMBER>-DS`
`<TEAM NUMBER>`
```

Reply:
I think this goes beyond the scope of the conversion.